frameworks/g-cloud-9/messages/contract_variation_1.yml

```diff
@@ -0,0 +1,466 @@
+variation_description_not_in_place: |
+    The Crown Commercial Service (CCS) has made some minor changes to the framework agreement and call-off contract (reference RM1557ix) because of the new General Data Protection Regulation (GDPR).<br/><br/>
```

Comment:
Comparing this to the g8 variation, on here we are including `br` tags whereas on the previous ones we didn't. I'm guessing they are nececssary?

Reply:
just a slight styling update